packages/ui/components/form/inputs/Input.tsx

```diff
@@ -338,6 +339,8 @@ const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLForm
           form
             .handleSubmit(handleSubmit)(event)
             .catch((err) => {
+              console.log(err);
+              // FIXME: Booking Pages don't have toast, so this error is never shown
```

Comment:
Can we dynamic load them? I don't want to increase booking pages bundle size any more.